_inc/client/components/multiple-choice-question/README.md

```diff
@@ -0,0 +1,157 @@
+<!-- @format -->
```

Comment:
❤️

Reply:
ha

Reply:
+1 ❤️ Once you adjust to auto-formatting nothing else will do.